src/cpu/aarch64/acl_convolution_utils.hpp

```diff
@@ -34,6 +34,7 @@ struct acl_obj_t {
     arm_compute::Tensor wei_tensor;
     arm_compute::Tensor bia_tensor;
     arm_compute::Tensor dst_tensor;
+    arm_compute::experimental::MemoryRequirements aux_mem_req;
```

Comment:
could you comment how using experimental feature from ACL can impact compatilibilty between ACL and oneDNN?
e.g. are there some backward compatibility guarantees?

Reply:
@mgouicem We made a new set of APIs specifically driven by oneDNN requirements and we expect stability but after we have all our planned patches in, targeted before the next oneDNN release. Things might change before that as our work gets internally and externally reviewed into `main`, but it seems unlikely, as we have most of the code ready and reviewed internally and the bottleneck now will be the speed of external reviews, and one more pending update to bump ACL up from 24.07 to 24.08.